docs/examples/model_beams.py

```diff
@@ -0,0 +1,47 @@
+from compas.files import OBJ
+from compas.datastructures import Mesh
+from compas_model.viewer import ViewerModel
+from compas_model.elements import Block
+from compas_model.model import Model
+from compas_model.algorithms import Collider
+
+obj = OBJ('data/beams3.obj')
+obj.read()
+meshes = []
+for name in obj.objects:
+    mesh = Mesh.from_vertices_and_faces(* obj.objects[name])
+    mesh.name = name
+    meshes.append(mesh)
+
+# from compas.files import STL
+# 
+
+# 
+# from compas.datastructures import Mesh
+
+# meshes = Mesh.from_stl('data/beams3.stl').exploded()
+
+# # Create elements from meshes and add them to the model.
```

Comment:
remove commented out parts (in all examples)

Reply:
Removed.